Source/DafnyLanguageServer/Handlers/DafnyHoverHandler.cs

```diff
@@ -20,7 +20,7 @@ public class DafnyHoverHandler : HoverHandlerBase {
     private readonly ILogger logger;
     private readonly IDocumentDatabase documents;
 
-    private const int RuLimitToBeOverCostly = 10000000;
+    public static int RuLimitToBeOverCostly = 10000000;
```

Comment:
I don't feel comfortable using mutable statics. Tests in separate files can be run concurrently and statics introduce the possibility that tests influence each other.